docs/quickstart/web.md

```diff
@@ -0,0 +1,53 @@
+---
+bodyclass: docs
+layout: docs
+headline: Web Quickstart
+sidenav: doc-side-quickstart-nav.html
+type: markdown
+---
+<p class="lead">This guide gets you started with gRPC-Web with a simple
+working example from the browser.</p>
+
+<div id="toc"></div>
+
+## Prerequisites
+
+* `docker` and `docker-compose`
+
+Please refer to [Docker website][] on how to install Docker.
+
+## Run an Echo example from the browser!
+
+```sh
+$ git clone https://github.com/grpc/grpc-web
+$ cd grpc-web
+$ docker-compose pull prereqs common echo-server envoy commonjs-client
+$ docker-compose up -d echo-server envoy commonjs-client
+```
+
+Open a browser tab, and go to:
+
+```
+http://localhost:8081/echotest.html
+```
+
+To shutdown, run `docker-compose down`.
+
+## Install the gRPC-Web protoc plugin
```

Comment:
This section seems out of place, If the gRPC-Web protoc plugin is covered in the basic tutorial, then we can remove this section from here.